src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -390,8 +390,7 @@ internal override ITypeSymbolInternal CommonGetWellKnownType(WellKnownType wellk
             ImmutableArray<KeyValuePair<WellKnownMember, TypedConstant>> namedArguments = default,
             bool isOptionalUse = false)
         {
-            DiagnosticInfo diagnosticInfo;
```

Comment:
>DiagnosticInfo diagnosticInfo; [](start = 12, length = 30)

Please check if there were changes around this in features/UsedAssemblyReferences branch. If there were, please revert this refactoring.